lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -85,12 +85,11 @@ def command_for_each_project(function):
 def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
     redcap_record_instances: List[REDCapRecord]) -> Optional[dict]:
 
-    assert redcap_record_instances is not None and len(redcap_record_instances) > 0, \
-        "The redcap_record_instances list was not populated."
-
```

Comment:
Do we need to log a warning and skip this record to replace this assert?

Reply:
I thought about that, but since we're already ensuring that we have 1 enrollment instance that means we are already covering the len > 0 case.
Do we need to cover the None case?
I don't think I needed this assert from the start.

Reply:
Hmmm, the list comprehension to create `enrollments` will error out if we get `redcap_record_instances` that is None.

Reply:
I didn't know if would ever be possible for the instances to be None, but I added that check/warning just in case.